runtime/darwinia/src/lib.rs

```diff
@@ -167,20 +168,26 @@ pub type Executive = frame_executive::Executive<
 	Block,
 	frame_system::ChainContext<Runtime>,
 	Runtime,
-	AllModules,
+	AllPallets,
 	CustomOnRuntimeUpgrade,
 >;
 /// The payload being signed in transactions.
 pub type SignedPayload = generic::SignedPayload<Call, SignedExtra>;
 
 type Ring = Balances;
 
+/// The BABE epoch configuration at genesis.
+pub const BABE_GENESIS_EPOCH_CONFIG: BabeEpochConfiguration = BabeEpochConfiguration {
+	c: PRIMARY_PROBABILITY,
+	allowed_slots: AllowedSlots::PrimaryAndSecondaryVRFSlots,
```

Comment:
This is wrong. The existing chain is running with `PrimaryAndSecondaryPlainSlots`, you should keep returning the same thing in `BabeApi::configuration()` as you were doing before.

Reply:
Thanks for the review & explanation